src/pages/NotFound/NotFound.test.js

```diff
@@ -1,10 +1,24 @@
 import React from "react";
-import { shallow } from "enzyme";
-import NotFound from "./NotFound";
+import configureStore from "redux-mock-store";
+import { Provider } from "react-redux";
+import { mount } from "enzyme";
+import { MemoryRouter } from "react-router";
+import dataDump from "testing/complete-redux-store-dump";
+
+import { Routes } from "components/Routes/Routes";
+
+const mockStore = configureStore([]);
 
 describe("NotFound page", () => {
-  it("renders without crashing and matches snapshot", () => {
-    const wrapper = shallow(<NotFound />);
-    expect(wrapper).toMatchSnapshot();
+  it("should display when unknown route is accessed", () => {
+    const store = mockStore(dataDump);
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter initialEntries={["/foobar11"]}>
+          <Routes />
+        </MemoryRouter>
+      </Provider>
+    );
+    expect(wrapper.find("NotFound").length).toBe(1);
```

Comment:
Could you also add a check to make sure nothing else was rendered by `Routes` that wasn't supposed to be?